plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/converter/GlueStatConverter.java

```diff
@@ -279,7 +279,7 @@ private static ColumnStatisticsData getGlueColumnStatisticsData(HiveColumnStatis
                 catalogColumnStatisticsData.setStringColumnStatisticsData(catalogStringData);
                 break;
             default:
-                throw new PrestoException(HIVE_INVALID_METADATA, "Invalid column statistics type: " + statistics);
+                throw new TrinoException(HIVE_INVALID_METADATA, "Invalid column statistics type: " + statistics);
```

Comment:
How does the `statistics` object look when logged?

Would it be more useful to just log `((PrimitiveTypeInfo) typeInfo).getPrimitiveCategory()`?